openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -87,14 +84,19 @@ static apr_status_t ssl_context_cleanup(void *data)
             free(c->password);
             c->password = NULL;
         }
+
+        tcn_atomic_uint32_free(&(c->ticket_keys_new));
```

Comment:
`&(c->ticket_keys_new)` -> `&c->ticket_keys_new` ? The parens shouldn't be necessary, right?